__tests__/app.test.js

```diff
@@ -301,3 +301,30 @@ describe("PATCH /api/articles/:article_id", () => {
     });
   });
 });
+describe("DELETE /api/comments/:comment_id", () => {
+  test("returns a 400 error for incorrect id type", async () => {
+    const response = await request(app)
+      .delete("/api/comments/banann")
+      .expect(400);
+    expect(response.body.msg).toBe("400 Bad Request");
+  });
+  test("returns a 404 error for non-existent id", async () => {
+    const response = await request(app).delete("/api/comments/678").expect(404);
+    expect(response.body.msg).toBe("404 comment_id of 678 Not Found");
```

Comment:
You should be less specific with error messages, for security reasons, people that are would use the api will have the endpoints documentation, so can figure stuff out that way, people that shouldn't be using it should get as little info as possible